src/containers/DebitCard/bridgeCard/transactionDetails.tsx

```
@@ -479,71 +842,297 @@ export default function TransactionDetails() {
+      result: 'base64',
+    });
+
+    const shareImage = {
+      title: t('SHARE_TITLE'),
+      message: `Hey! I just spent ${transaction.amount} at ${transaction.metadata?.merchant} using my Cypher Card! 🚀 Living the crypto life!`,
+      subject: t('SHARE_TITLE'),
+      url: `data:image/jpeg;base64,${url}`,
+    };
+
+    if (!isAndroid()) {
+      delete shareImage.message;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)